editor/src/app/sites/sections/entries/entries-state/section-entries.state.ts

```diff
@@ -22,6 +24,22 @@ export class SectionEntriesState implements NgxsOnInit {
     });
   }
 
+  @Action(AddSectionEntriesAction)
+  addSectionEntries({ patchState, getState }: StateContext<SectionEntriesStateModel>, action: AddSectionEntriesAction) {
+    const state = getState();
+    const newEntries = {};
+    newEntries[action.siteName] = state[action.siteName].concat(action.entries);
```

Comment:
we  can use the spred operator to simplify this - `[...state[action.siteName], ...action.entries]`